backend/src/file/services.py

```diff
@@ -0,0 +1,26 @@
+import magic
+from fastapi import HTTPException, Response
+
+from src.core.cloudstorage import CloudStorage
+
+
+def guess_media_type(data: bytes) -> str:
+    mime = magic.Magic(mime=True)
+    return mime.from_buffer(data)
```

Comment:
I would give here also little docstring explaining what we are doing, its not that clear for everybody that we are checking media type by bytes